source/braille.py

```diff
@@ -546,7 +551,14 @@ def getPropertiesBraille(**propertyValues) -> str:  # noqa: C901
 		textList.append(description)
 	hasDetails = propertyValues.get("hasDetails")
 	if hasDetails:
-		textList.append("details")
+		detailsRole = propertyValues.get("detailsRole", controlTypes.Role.UNKNOWN)
+		if detailsRole != controlTypes.Role.UNKNOWN:
+			textList.append(roleLabels.get(detailsRole, detailsRole.displayString))

```

Comment:
Should this be something like "has cmnt", "has sggstn", etc instead?

Reply:
Do those roles have a shorthand labels already? You can potentially combine them programmatically.

Reply:
Yes they do - the current behaviour is to just report "cmnt", "fnote", "details" as opposed to "has details"/"has fnote"

Reply:
@michaelDCurran / @leonardder / any other braille users.
When navigating an object with details, the current behaviour is to report "details" for braille and "has details" for speech. 
When navigating an object with a known type of details, the behaviour for speech is to report "has comment".

Reporting "cmnt" for braille is misleading. Is "has cmnt" appropriate for braille? Is there a shorter alternative?

Reply:
No - but in absence of a clear answer, "has cmnt" has been suggested by @michaelDCurran to be the best option.

Reply:
I'm editing the description to describe the outcome and point to this conversation.